src/server/middleware/adminRoutes.ts

```diff
@@ -37,12 +37,10 @@ const QUEUES: Queue[] = [
 export const withAdminRoutes = async (fastify: FastifyInstance) => {
   // Configure basic auth.
   await fastify.register(fastifyBasicAuth, {
-    validate: (username, password, req, reply, done) => {
```

Comment:
Updating all fastify "callback style" or "async/await" style. This will be strictly enforced in a future fastify version.